xbmc/video/dialogs/GUIDialogVideoVersion.h

```diff
@@ -0,0 +1,58 @@
+/*
+ *  Copyright (C) 2005-2018 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
```

Comment:
Note, when adding new windows/dialogs it needs to be added to doxy too: https://github.com/xbmc/xbmc/blob/master/xbmc/guilib/WindowIDs.dox

Reply:
Will do.